usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -207,6 +207,13 @@ def get_or_create_program_activity(row, submission_attributes):
     if prg_activity is None and row['program_activity_code'] is not None:
         prg_activity = RefProgramActivity.objects.create(**filters)
         logger.warning('Created missing program activity record for {}'.format(str(filters)))
+    # If the PA has a blank name, update it with the value in the row.
+    # PA loader should overwrite the names for the unique PAs from the official
+    # domain values list if the title needs updating, but for now grab it from the submission
+    if prg_activity and not prg_activity.program_activity_name:
```

Comment:
Yeah, definitely we were not putting in program activity name when creating PA records on the fly, so not surprising that you saw blank names. What I'm trying to figure out is why we can't just pop in the name when creating a new PA record up in lines 207-8. Does that make sense?